core/src/main/kotlin/org/evomaster/core/output/clustering/DBSCANClusterer.kt

```diff
@@ -0,0 +1,137 @@
+package org.evomaster.core.output.clustering
+
+import org.evomaster.core.output.clustering.metrics.DistanceMetric
+
+/**
+ * Implementation of density-based clustering algorithm DBSCAN.
+ *
+ * Original Publication:
+ * Ester, Martin; Kriegel, Hans-Peter; Sander, J�rg; Xu, Xiaowei (1996).
+ * Simoudis, Evangelos; Han, Jiawei; Fayyad, Usama M., eds.
```

Comment:
ah, the beauty of copy&paste a text from encoding ISO-8859-1 into a UTF-8 Kotlin file :)